script/nnbd_plugins.sh

```diff
@@ -29,7 +30,7 @@ readonly NNBD_PLUGINS_LIST=(
 
 readonly NON_NNBD_PLUGINS_LIST=(
   # "android_alarm_manager"
-  "camera"
+  # "camera"
```

Comment:
@mvanbeusekom I think the issue might be that `camera` and `google_maps_flutter` now conflict because of `stream_transform`. I think uncommenting `google_maps_flutter` in this list might fix the build errors.

What are your thoughts @ditman?

Reply:
So it looks like camera also has a problem with `google_sign_in` on master:

```
ERR : Because every version of google_sign_in_platform_interface from path depends on quiver ^3.0.0-nullsafety.2 and every version of camera from path depends on quiver ^2.1.5, google_sign_in_platform_interface from path is incompatible with camera from path.

    | So, because all_plugins depends on both camera from path and google_sign_in_platform_interface from path, version solving failed.

FINE: Exception type: SolveFailure
```

I was able to get it to build on master locally when keeping `camera` uncommented and `google_maps_flutter" commented. I think `camera` is going to have to stay in this list until the `camera` package is migrated.